engine/src/main/scala/cromwell/engine/package.scala

```diff
@@ -87,5 +87,6 @@ package object engine {
                                             failureMode: WorkflowFailureMode) {
     def id = backendDescriptor.id
     def namespace = backendDescriptor.workflowNamespace
+    def name = namespace.workflow.unqualifiedName
```

Comment:
I realize that this class still isn't the worst offender in the package object but can you pull it out into its own thing? It looks like we need to clean out our package objects again.